app/internal/security/dependancies.py

```diff
@@ -21,5 +25,32 @@ async def is_manager(
     """
     A dependency function protecting routes for only logged in manager
     """
-    await check_jwt_token(db, jwt, manager=True)
-    return True
+    jwt_payload = await get_jwt_token(db, jwt)
+    if jwt_payload.get("is_manager"):
+        return True
+    raise HTTPException(
+                status_code=HTTP_401_UNAUTHORIZED,
+                headers=request.url.path,
+                detail="You don't have a permition to enter this page")
+
+
+async def current_user(
+    request: Request,
+    db: Session = Depends(get_db),
+    jwt: str = Depends(get_authorization_cookie),
+) -> User:
+    """
+    Returns logged in User object.
+    A dependency function protecting routes for only logged in user.
+    """
+    jwt_payload = await get_jwt_token(db, jwt)
+    username = jwt_payload.get("sub")
+    user_id = jwt_payload.get("user_id")
+    db_user = await User.get_by_username(db, username=username)
+    if db_user and db_user.id == user_id:
+        return db_user
```

Comment:
Prefer to returning a dictionary with the user's details here.
We don't want to inquire the DB every time someone wants to use the current_user dependency :)

You can also create another function, `current_user_from_db` to supply more options